Code/Engine/RendererCore/Debug/Implementation/DebugRenderer.cpp

```diff
@@ -102,7 +103,7 @@ namespace
     }
 
     ezUInt64 m_uiLastRenderedFrame;
-    PerContextData* m_pData[2];
+    ezUniquePtr<PerContextData> m_pData[2];
```

Comment:
This was a memory leak, and when you used one of the 'textured' calls (e.g. the textured Draw2DRectangle variant), D3D would complain about live GPU objects at shutdown.

Reply:
Indeed, good catch!